layers/core/app/utils/shiki.ts

```diff
@@ -0,0 +1,23 @@
+import { getHighlighter } from 'shiki'
```

Comment:
We should use a lazy imported of `shikiji`. And `nuxt/mdc` module already exposes an endpoint for it.